lib/mongoose_push_web/schemas/request/send_notification/deep/alert.ex

```diff
@@ -0,0 +1,34 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep.Alert do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Request.SendNotification.Deep.Alert",
+    description: "In this request one can pass alert field only.",
+    type: :object,
+    allOf: [MongoosePushWeb.Schemas.Request.SendNotification.Deep],
+    properties: %{
+      alert: %Schema{
+        type: :object,
+        properties: %{
```

Comment:
According to documentation, this object should be one of the childs of `allOf`: https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/. Right now we require this schema to be parsed agains `allOf` `[MongoosePushWeb.Schemas.Request.SendNotification.Deep]` which is equivalent of just using `MongoosePushWeb.Schemas.Request.SendNotification.Deep` schema.

Reply:
You're right and I corrected that also in `Data` module.